<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LambdaSnail::Juno::expenses::ExpensesOverviewWidget</class>
 <widget class="QWidget" name="LambdaSnail::Juno::expenses::ExpensesOverviewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExpensesOverviewWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBar" name="toolBar"/>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
